src/main/java/fruitshop/strategy/PurchaseOperation.java

```diff
@@ -0,0 +1,26 @@
+package fruitshop.strategy;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.Storage;
+
+public class PurchaseOperation implements OperationHandler {
+    public static final String READ_FILE_PATH = "src/main/resources/reportToRead.csv";
```

Comment:
Avoid hardcoding file paths in your classes. This path should be passed as a parameter to the method that needs it, to avoid coupling your class with a specific file location. Consider removing this constant and refactoring the code to accept the file path as a method parameter or through constructor injection.